### PR TITLE
Do not use CSC Bad Chambers list during muon reconstruction

### PR DESCRIPTION
**Pull request by jhgoh**
CSC Bad chamber list was used in the muon reconstruction by default but this information but using this information is not recommended by the CSC DPG.
Excluding bad chamber may cause unexpected inefficiency if the information is not up to date.

The feature to exclude 'bad' CSCs are turned off by changing the default parameter in cff file.

### CODE REVIEW

File: TrackingTools/TrackAssociator/python/DetIdAssociatorESProducer_cff.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     etaBinSize = cms.double(0.125),
     nEta = cms.int32(48),
     nPhi = cms.int32(48),
-    includeBadChambers = cms.bool(False)
+    includeBadChambers = cms.bool(True)
```
Comment (slava77) [inline TrackingTools/TrackAssociator/python/DetIdAssociatorESProducer_cff.py]: a change made globally in this file will introduce problems in run1 simulation.
Run-2 specific changes should be added in postLS1 customizations

Comment (slava77) [inline TrackingTools/TrackAssociator/python/DetIdAssociatorESProducer_cff.py]: ... maybe we actually add this to Configuration/StandardSequences/python/Reconstruction_Data_cff.py ?
I think it's more appropriate than even doing it just for run2

**Comment by jhgoh** [comment]
@slava77 includeBadChamber parameter is changed in the PostLS1 customization.

**Comment by slava77** [comment]
@cmsbuild please test
(the one sent from email client didn't make it )

**Comment by civanch** [comment]
+1
however, the name of the parameter is misleading - these chambers are not "bad", I would change this name

**Comment by slava77** [comment]
@davidlange6 
you mentioned on Tuesday that you will have some inputs on this PR.

**Comment by slava77** [comment]
hold

we should update RecoTLR.py file to use these postLS1 customs directly.
This was on my to-do list for some time, but wasn't done yet.
Current PR has no effect on data.

**Comment by slava77** [comment]
@jhgoh during the meeting today I was suggesting to move the modification to the ID to postLS1Customs.py function customise_Reco.
This function is picked up in T0 processing as well and you will not need to wait for my modifications there.

**Comment by jhgoh** [comment]
Done.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
- Looking at jenkins outputs: there are no changes as expected
    - run1 setup is not affected by construction
    - run2 MC has no bad chambers, so nothing to skip before the change
- tested locally in CMSSW_7_6_0_pre5 on 200 events in 251721, 256729, 254790: there is a change in an expected place the number of crossed chambers by the muon increases (1 muon that turned out to be in DoubleEG251721). There is no effect on other monitored quantities
wfdoubleeg251721_nchambers_norpc

**Comment by slava77** [comment]
"+1" (even though it's not a proper cms-bot signature)

**Comment by davidlange6** [comment]
+1

**Comment by civanch** [comment]
+1

**Pull request merged**